src/StateService/StatePlugin.cs

```diff
@@ -80,6 +80,7 @@ void IPersistencePlugin.OnPersist(NeoSystem system, Block block, DataCache snaps
         [ConsoleCommand("start states", Category = "StateService", Description = "Start as a state verifier if wallet is open")]
         private void OnStartVerifyingState()
         {
+            if (System.Settings.Magic != Settings.Default.Network) throw new InvalidOperationException("Network doesn't match");
```

Comment:
```suggestion
            if (system.Settings.Magic != Settings.Default.Network) throw new InvalidOperationException("Network doesn't match");
```

Reply:
There is no field named `system`;
I add `System` check.